### PR TITLE
Bug fix: fix the wrong modifier for windows.

### PR DESCRIPTION
**Pull request by shaoshing**
Bug fix for #1730
@chadrolfs

### CODE REVIEW

**Comment by chadrolfs** [comment]
@shaoshing this fixes the load on windows issue, however on this branch, ctrl++ (numpad) doesn't work.

**Comment by shaoshing** [comment]
Ah, I kept forgetting the windows config. I just updated its menu shortcut, now it should work. Sorry for the back and forth.

**Comment by chadrolfs** [comment]
No worries, that's part of the game.  :grinning:  Thanks for jumping on this!

**Comment by chadrolfs** [comment]
confirmed fix and shortcuts working on Win

**Comment by shaoshing** [comment]
:+1:  Thank you

File: src/js/actions/ui.js [hunk 1]
```diff
@@ -412,7 +413,8 @@ define(function (require, exports) {
         var pathPromise = adapterUI.setSuppressTargetPaths(true);
 
         // Add additional shortcut CMD=, so that CMD+ and CMD= both work for zoom in.
-        var zoomInShortcutPromise = this.transfer(shortcuts.addShortcut, "=", { "command": true }, function () {
+        var zoomShortcutModifier = system.isMac ? { "command": true } : { "control": true };
+        var zoomInShortcutPromise = this.transfer(shortcuts.addShortcut, "=", zoomShortcutModifier, function () {
```
Comment (volfied) [inline src/js/actions/ui.js]: These var declarations should either be comma separated, or new line separated.

Also a newline between 417 and 418 would make this code more readable.

**Comment by volfied** [comment]
:space_invader: Nits!

**Comment by volfied** [comment]
:+1:

**Pull request merged**